project.clj

```diff
@@ -1,4 +1,4 @@
-(def tk-version "1.3.0")
+(def tk-version "1.3.1-SNAPSHOT")
```

Comment:
It looks like https://github.com/puppetlabs/trapperkeeper/pull/221 has been merged, so we just need a TK release.